index.html

```diff
@@ -0,0 +1,134 @@
+          <div class="logo-box"><span>Art</span>~Blog</div>
+          <nav class="header-nav-container__nav-bar">
+            <div class="header-nav-container__nav-bar--icon" id="hamberger">
+              <svg class="header-nav-container__nav-bar--icon" id="icon-menu">
+                <use xlink:href="assets/Icons/sprite.svg#icon-menu"></use>
+              </svg>
+              <svg class="header-nav-container__nav-bar--icon" id="icon-cross">
+                <use xlink:href="assets/Icons/sprite.svg#icon-cross"></use>
+              </svg>
+            </div>
+            <ul class="header-nav-container__nav-bar--link">
+              <li class="nav-link"><a href="pages/about.html">About</a></li>
+              <li class="nav-link"><a href="">Program</a></li>
+              <li class="nav-link"><a href="">Join</a></li>
+              <li class="nav-link"><a href="">Sponsor</a></li>
+              <li class="nav-link"><a href="">News</a></li>
+            </ul>
+            <button class="your-blog">Your Blog</button>
+          </nav>
+        </div>
+      </section>
+
+      <!-- Header Body -->
+      <section class="header-body">
+        <p class="header-body__greeting-text">"Hello! Share your art"</p>
+        <h1 class="header-body__header-title">
+          <span>Creativity in Action</span>
+          <span>ArtBlog 2022</span>
+        </h1>
+        <div class="header-body__header-desc">
+          <p>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Doloremque beatae accusamus tempora voluptatibus
+            laudantium dolor molestias suscipit, ratione tenetur, delectus harum a perspiciatis consequuntur, totam non.
+            Aperiam deleniti illo dolorem!</p>
+        </div>
+        <p class="header-body__event-time">2022.03.16(WED) ~ 18(FRI)</p>
+        <p class="header-body__event-place">@ National Museum of Korea, Art Center Nabi and More</p>
+      </section>
+    </header>
+
+    <!-- Main Section -->
+    <main class="main">
+
+      <!-- Art Section -->
+      <section class="main-art">
+        <div class="main-container">
+          <h2 class="main-container__main-title">Main Arts</h2>
+          <div class="main-container__art"></div>
+          <a href="" class="main-container__detail">see the whole arts</a>
+        </div>
+      </section>
+
+      <!-- Featured Speakers-->
+      <section class="featured-speakers"></section>
+    </main>
+
+    <!-- Footer Section -->
+    <footer class="footer">
+      <div class="footer__main">
+        <div class="footer__main--content">
+          <h2 class="footer__main--title">Partners</h2>
+          <div class="footer__main--partners">
+            <div class="partner">Pexels</div>
+            <div class="partner">HP</div>
+            <div class="partner">Amazon</div>
+            <div class="partner">Mozilla</div>
+            <div class="partner">Google</div>
+            <div class="partner">Art ~ Blog</div>
+          </div>
```

Comment:
- [ ] Since this is a list of items, It would be more semantically correct if you used `li` tags wrapped in a `ul`